Readme.md

```diff
@@ -3,16 +3,29 @@
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [PdfFormFilling.cs](./CS/PdfFormFilling/PdfFormFilling.cs) (VB: [PdfFormFilling.vb](./VB/PdfFormFilling/PdfFormFilling.vb))
-<!-- default file list end -->
-# PDF Document API - Use PdfFormData to Fill Interactive Form Fields
+# PDF Document API - Fill Interactive Form Fields
+
+This example demonstrates how to retrieve a list of form field names and specify field values of an interactive form.
+
+> You need a license for the [DevExpress Office File API Subscription](https://www.devexpress.com/products/net/office-file-api/) or [DevExpress Universal Subscription](https://www.devexpress.com/subscriptions/universal.xml) to use this library in production code. 
 
+## Implementation Details
 
-The example shows how to fill an interactive form using field names. <br><br>To learn how to get a checked appearance name for the check box, see the <a href="https://documentation.devexpress.com/OfficeFileAPI/120047/PDF-Document-API/Examples/Interactive-Form/How-to-Obtain-a-Checked-Appearance-Name-for-a-Check-Box">How to: Obtain a Checked Appearance Name for a Check Box </a>example.<br><br>To learn how to get a checked appearance name for the radio button, see <a href="https://documentation.devexpress.com/OfficeFileAPI/120190/PDF-Document-API/Examples/Interactive-Form/How-to-Obtain-a-Checked-Appearance-Name-for-Each-Radio-Button-in-the-Radio-Group">How to: Obtain a Checked Appearance Name for Each Radio Button in the Radio Group</a> example.<br><br>The Universal Subscription or an additional Office File API Subscription is required to use this example in production code. Please refer to the <a href="https://www.devexpress.com/Subscriptions/">DevExpress Subscription</a> page for pricing information. <br><br>
+1. Call the [PdfDocumentProcessor.LoadDocument](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentProcessor.LoadDocument.overloads) method to load a PDF document with an interactive form. 
+2. Call the [PdfDocumentProcessor.GetFormData](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentProcessor.GetFormData) method to retrieve the [PdfFormData](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfFormData) object that contains interactive form data. 
+3. Specify field values of an interactive form. Call the [PdfDocumentProcessor.ApplyFormData](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentProcessor.ApplyFormData(DevExpress.Pdf.PdfFormData)) method to apply data to the interactive form and save the PDF document.
+4. You can call the [PdfDocumentProcessor.GetFormFieldNames](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentProcessor.GetFormFieldNames) method to retrieve a list of field names and iterate through the returned string collection.
```

Comment:
It seems strange to start all these sentences with verbs, but then 'You can'?

Reply:
Can I start this sentence from 'call' as follows: Call the [PdfDocumentProcessor.GetFormFieldNames](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentProcessor.GetFormFieldNames) method to retrieve a list of field names and iterate through the returned string collection.

Reply:
Ok